alerter.c

```diff
@@ -27,6 +33,9 @@ int main() {
     alertInCelcius(400.5);

```

Comment:
Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold.

Reply:
Updated